asteroid_definitions/solar-system.lua

```diff
@@ -14,19 +16,78 @@ local Uranus = require("asteroid_definitions.uranus")
 local Neptune = require("asteroid_definitions.neptune")
 local Pluto = require("asteroid_definitions.pluto")
 local AsteroidBeltOuter = require("asteroid_definitions.asteroid-belt-outer")
+local MakeMake = require("asteroid_definitions.makemake")
+local Eris = require("asteroid_definitions.eris")
+local Haumea = require("asteroid_definitions.haumea")
+local GongGong = require("asteroid_definitions.gonggong")
```

Comment:
These 4 objects (inside the Kuiper belt) were added in the forked repo and are now added here.